pkg/storage/snapshot/restore_test.go

```diff
@@ -1398,7 +1407,13 @@ func expectPVCCreateWithDataSourceRef(client *k8sfake.Clientset, vmRestore *snap
 
 		Expect(createObj.Spec.DataSource).ToNot(BeNil())
 		Expect(createObj.Spec.DataSourceRef).ToNot(BeNil())
-		Expect(createObj.Spec.DataSource).To(Equal(createObj.Spec.DataSourceRef))
+
+		dataSourceRef := &corev1.TypedLocalObjectReference{
```

Comment:
nit: `dataSource` instead of `dataSourceRef`

Reply:
Well, intricate :) You are right, since both `Spec.DataSource` and `Spec.DataSourceRef` are references. My reasoning was mainly based on the fact that you are then going to compare it with `Spec.DataSource`.
It can stay as is from my point of view